src/components/capture/creation/DetailsForm.tsx

```diff
@@ -22,12 +22,9 @@ function NewCaptureDetails(props: NewCaptureDetailsProps) {
     const intl = useIntl();
     const { state, dispatch, readonly, displayValidation } = props;
 
-    const {
-        data: { connectors },
-        loading,
-        error,
-    } = useConnectors();
+    const { data: connectorsData, isError, isSuccess, error } = useConnectors();
 
+    const [isPostProcessingDone, setPostProcessingDone] = useState(false);
```

Comment:
One pattern for handling post processing.... there is another in here as well. We'll eventually figure out what one we like better.